### PR TITLE
Arome / Partners page update

### PR DESCRIPTION
**Pull request by arome-binary**
Changes:

-  Updated partners page

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by ShuvoHabib — CHANGES_REQUESTED** [review]

File: src/components/layout/nav.js [hunk 1]
```diff
@@ -938,6 +938,16 @@ export const NavPartners = ({ no_login_signup }) => {
                                     {localize('Payment agents')}
                                 </StyledLink>
                             </NavLink>
+                            <NavLink>
+                                <StyledLink
+                                    active={current_page === 'developers'}
+                                    activeClassName="active"
+                                    to="https://developers.deriv.com"
```
[the remark this answers is not in the document]
Comment (arome-binary) [reply src/components/layout/nav.js:945]: Noted @ShuvoHabib , checking that out.

File: src/components/layout/footer.js [hunk 2]
```diff
@@ -474,6 +474,11 @@ const Footer = ({ type, is_ppc, is_ppc_redirect }) => {
                                             {localize('Payment agents')}
                                         </Link>
                                     </LinkWrapper>
+                                    <LinkWrapper>
+                                        <Link to="http://developers.deriv.com/">
```
[the remark this answers is not in the document]
Comment (arome-binary) [reply src/components/layout/footer.js:478]: Noted @ShuvoHabib , checking that out.

File: src/pages/trader-tools/_tools.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    margin-left: ${(props) => props.margin_left};
+
+    @media ${device.tabletL} {
+        max-width: 100%;
+        margin: 0;
+    }
+`
+const StyledLinkButton = styled(LinkButton)`
+    margin-top: 16px;
+    width: fit-content;
```
[the remark this answers is not in the document]
Comment (arome-binary) [reply src/pages/trader-tools/_tools.js:51]: Noted @ShuvoHabib , checking that out.

File: src/common/utility.js [hunk 4]
```diff
@@ -211,61 +213,62 @@ const getDomain = () =>
...
-    dmt5_windows_url,
-    dmt5_linux_url,
-    dmt5_android_url,
-    dmt5_ios_url,
     community_url,
+    debounce,
     deriv_app_url,
     deriv_blog_url,
     deriv_bot_app_url,
+    deriv_cookie_domain,
```
Comment (ShuvoHabib) [inline src/common/utility.js:231]: If all the conflicts are fixed, you can see, constants and utility functions are separated. You need import them from respected files.

File: src/pages/trader-tools/_tools.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (arome-binary) [reply src/pages/trader-tools/_tools.js:51]: https://github.com/binary-com/deriv-com/pull/1668

File: src/components/layout/nav.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (arome-binary) [reply src/components/layout/nav.js:945]: https://github.com/binary-com/deriv-com/pull/1668

File: src/common/utility.js (hunk 4 above)
Comment (arome-binary) [reply src/common/utility.js:231]: Resolved here https://github.com/binary-com/deriv-com/pull/1668

**Pull request closed without merging**